examples/determinator-rails/Procfile

```diff
@@ -1,2 +1 @@
 web: bundle exec rails s
-worker: bundle exec sidekiq
```

Comment:
Will the same app still actually work? Don't we need _some_ routemaster processor to operate this sample app?

Reply:
The cache busting endpoint doesn't require any jobs. Thought it'd be simpler.